src/app/chat/views/activity_notification_list.nim

```diff
@@ -135,6 +135,27 @@ QtObject:
       self.nbUnreadNotifications = 0
     self.unreadCountChanged()
 
+  proc markActivityCenterNotificationUnread(self: ActivityNotificationList, notificationId: string,
+  communityId: string, channelId: string, nType: int): void {.slot.} =
+    let notificationType = ActivityCenterNotificationType(nType)
+    let markAsUnreadProps = MarkAsUnreadNotificationProperties(communityId: communityId,
+    channelId: channelId, notificationTypes: @[notificationType])
+
+    let error = self.status.chat.markActivityCenterNotificationUnread(notificationId, markAsUnreadProps)
+    if (error != ""):
+      return
```

Comment:
Should we return the error? Or maybe just log it?

Reply:
I was kinda inspired by the `markActivityCenterNotificationRead` API which is very similar.

So I think we either leave as is, or I should also change that API to log the error.